src/model-mixin.vue

```diff
@@ -503,6 +519,12 @@ export default {
 
             this.renderer.render( this.scene, this.camera )
 
+            if ( this.debug ) {
+                window.$renderer = this.renderer;
+                window.$camera = this.camera;
+                window.$scene = this.scene;
```

Comment:
This code should not be in render(), because there is no need to repeat assignment.

Reply:
Ok, it was just an idea to have access to the variables using the developer console because there is no other way to get access to these variables as far as I know.